pkg/collector/pipeline.go

```diff
@@ -117,3 +165,11 @@ func (p *Pipeline) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 	ctx := r.Context()
 	p.ProcessReports(ctx, w, r)
 }
+
+// Close stops the processing, such that anything in the queue
```

Comment:
I'd reword this to make it clear that it's the responsibility of the caller to make sure no one else tries to add more reports after calling this method. If I'm reading correctly, the program will panic if that happens (by sending to a closed channel).

Reply:
Done